setup.py

```diff
@@ -37,8 +37,10 @@
 if 'SKIP_PYTHON_MODULES' in os.environ:
     kwargs['packages'] = []
     kwargs['package_dir'] = {}
+    kwargs['install_requires'].remove('catkin_pkg')
```

Comment:
Is there a reason this is necessary? I was under the impression that the python3-catkin-pkg dependency would only be inserted if it wasn't already in the list. If the rospkg scripts package doesn't depend on catkin-pkg at all then shouldn't it also be removed from the stdeb.cfg? https://github.com/ros-infrastructure/rospkg/blob/5174b0708912f81b6962b2f86a3ddd611542ee46/stdeb.cfg#L6-L10

Reply:
> shouldn't it also be removed from the stdeb.cfg?

Indeed it should: c70ab63b34650959fe7db503f11c514efe4b5a9f